FIAT/reference_element.py

```diff
@@ -1025,6 +1025,28 @@ def tuple_sum(tree):
         return tree
 
 
+def is_hypercube(cell):
+    if isinstance(cell, TensorProductCell):
+        return reduce(lambda a, b: a and b, [is_hypercube(c) for c in cell.cells])
+    else:
+        return isinstance(cell, (DefaultLine, UFCInterval, UFCQuadrilateral, UFCHexahedron))
+
+
+def flatten_reference_cube(ref_el):
+    """ ... """
```

Comment:
You copied my placeholder docstring. You should actually give a description of what this function is actually doing

Reply:
The docstring you had in a previous commit was just fine!